### DISCUSSION TITLE
Pairing with twin.macro allows only Tailwind classes to be folded

### DISCUSSION THREAD

**Discussion by colemilne54** [moalamri/vscode-inline-fold#117] [Show and tell]
Pairing VSCode inline fold with twin.macro allows you to fold Tailwind classes while keeping your custom classes visible.

[Demo/ How-to](https://github.com/colemilne54/twin-fold)

**Comment by emedeirox** [comment]
@colemilne54 This is a nice use case, thank you very much!

add this to your settings.json.
```yaml
    "inlineFold.regexGroup": "6"
```
my global regexGroup is 7 so it don't fold well, with the repo. may affect some more people.

And feel free to add a pull request to the docs with this use case. I'm glad to know there such a thing.
